src/components/location-button/location-button.component.tsx

```diff
@@ -15,7 +15,7 @@ export interface IProps {
 }
 
 /**
- * Touchable component which triggers the onBuildingLocationPress
+ * Touchable component which triggers the onLocationButtonPress
```

Comment:
Instead of onLocationButtonPress, it should be set to ```setUserCurrentLocation``` as it was changed in US-5.

Reply:
Also addressed here:
https://github.com/RGPosadas/WayFinder/pull/103#discussion_r407136091